bin/gp.sh

```diff
@@ -11,7 +11,7 @@ branch=${3:-"master"} # default to master when branch isn't specified
 
 # make folder (same as input, no checking!)
 mkdir $repo
-git clone git@github.com:$org/$repo.git --single-branch
+git clone http://github.com/$org/$repo.git --single-branch
```

Comment:
This should probably be "https". GitHub redirects to https anyway, so that will save the script a redirect.